kasparovsyncd/mqtt/block.go

```diff
@@ -0,0 +1,38 @@
+package mqtt
+
+import (
+	"github.com/kaspanet/kasparov/apimodels"
+	"github.com/kaspanet/kasparov/dbaccess"
+	"github.com/kaspanet/kasparov/dbmodels"
+)
+
+// BlockAddedTopic is an MQTT topic for new blocks
+const BlockAddedTopic = "block-added"
+
+// PublishBlockAddedNotifications publishes notifications for the block
+// that was added, and notifications for its transactions.
+func PublishBlockAddedNotifications(hash string) error {
+	if !isConnected() {
+		return nil
+	}
+
+	preloadedFields := make([]dbmodels.FieldName, len(dbmodels.TransactionRecommendedPreloadedFields)+1)
```

Comment:
Please export the part that appends `Transaction.` to all dbmodels.TransactionRecommendedPreloadedFields to function/constant in dbmodels package.